tests/Provider/Doctrine/ORM/DateIntervalHelperTest.php

```diff
@@ -15,7 +15,8 @@ public function testYearsRejectsInvalidValue($years)
     {
         $helper = new DateIntervalHelper(new \DateTime());
         
-        $this->setExpectedException('InvalidArgumentException', sprintf(
+        $this->expectException('InvalidArgumentException');
```

Comment:
Can you use `\InvalidArgumentException::class`?

Reply:
Absolutely!